azkaban-common/src/main/java/azkaban/imagemgmt/permission/PermissionManagerImpl.java

```diff
@@ -76,4 +83,24 @@ public boolean hasPermission(final String imageTypeName, final String userId, fi
 
     return hasPermission;
   }
+
+  /**
+   * Checks if the user has permission to manage high priority flows.
+   * @param userId : must not be null.
+   * @return boolean
+   */
+  @Override
+  public boolean hasPermission(final String userId) {
```

Comment:
It seems all HP owners have permission to update the hp_flows table.

Reply:
In this PR, yes. Eventually no.